libc/newhdrgen/yaml/assert.yaml

```diff
@@ -0,0 +1,17 @@
+header: assert.h
+macros: []
+types: []
+enums: []
+objects: []
+functions:
+  - name: __assert_fail 
+    standards:
+      - llvm_libc_ext
+    return_type: _Noreturn void
+    arguments:
+      - type: const char *
+      - type: const char *
+      - type: unsigned
+      - type: const char *
+    guard: __cplusplus
+  
```

Comment:
Remove extra line here, but otherwise LGTM

Reply:
In the other Yaml files all of them have a newline at the end as well.

Reply:
If there are extra spaces on the newline, github will complain about it. If line is empty the line number should turn gray.